app/routes.py

```diff
@@ -1,2 +1,188 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, make_response, jsonify
+from datetime import datetime
+import requests
+import flask_migrate
+import os
+from app.models.goal import Goal
+
+
+
+task_bp = Blueprint("task", __name__, url_prefix='/tasks')
+goal_bp = Blueprint("goal", __name__, url_prefix='/goals')
+
+@task_bp.route("", methods=["POST"], strict_slashes = False)
+def post_task():
+    request_body = request.get_json()
+
+    if "title" in request_body and "description" in request_body and "completed_at" in request_body:
+        new_task = Task(title=request_body["title"],description=request_body["description"], 
+                    completed_at=request_body["completed_at"])
+        db.session.add(new_task)
+        db.session.commit()
+        return jsonify({"task": new_task.to_json()}), 201
+    else: 
+        return make_response({"details": "Invalid data"}), 400     
```

Comment:
Consider refactoring to handle the 400 response first. This pattern can increase readability.

```suggestion
    if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
        return make_response({"details": "Invalid data"}), 400 
    new_task=Task(title=request_body["title"],description=request_body["description"], 
    completed_at=request_body["completed_at"])
    db.session.add(new_task)
    db.session.commit()
    return jsonify({"task": new_task.to_json()}), 201
```